koap/js_tests/package.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
I messed around a lot in this file trying to get importing working. There's some stuff I probably don't need.

Reply:
Everything in here should be needed now.